lib/improver/psychrometric_calculations/psychrometric_calculations.py

```diff
@@ -583,6 +583,8 @@ def process(self, wet_bulb_temperature):
             wet_bulb_temperature_integral (iris.cube.Cube):
                 Cube of wet bulb temperature integral (Kelvin-metres).
         """
+        # Touch the data to ensure it is not lazy
```

Comment:
```suggestion
        # Touch the data to ensure it is not lazy
        # otherwise vertical interpolation is slow
```

Reply:
@bayliffe thanks for the clarification - updated the comment so I don't forget why we needed this